Services/data/devicemodels/scripts/engine-02-state.js

```diff
@@ -79,6 +79,8 @@ function main(context, previousState, previousProperties) {
     // 10 +/- 5%,  Min 0, Max 20
     if (state.fuellevel > 0) {
         state.vibration = vary(10, 5, 0, 20);
+    } else {
+        state.vibration = 0;
```

Comment:
If the device is out of fuel, it's movement will stop and shouldn't be vibrating.